SI/Sem.03/Pract.03/Таsк06.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+int main() {
+	int num = 0, count = 0;
+	do {//checks if the number is possitive or 0
+		std::cin >> num;
+		if (num < 0) {
+			std::cout << "Insert positive number: " << std::endl;
+		}
+		if (num == 0) break;
```

Comment:
You do not need a break right before the end of the body - if num <= 0, the condition on line 10 will break the loop. That is why you do not need any other if-statements.